mpa/cls/trainer.py

```diff
@@ -117,16 +117,18 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         # cfg.dump(osp.join(cfg.work_dir, 'config.yaml')) # FIXME bug to save
         # logger.info(f'Config:\n{cfg.pretty_text}')
 
+        # TODO (sungchul): check other condition for vaildation
+        validate = False if cfg.model.type in ['BYOL'] else True
```

Comment:
Could you explain why BYOL has different value with other algorithms?

Reply:
Most models not using warmstart probably have `SAMImageClassifier` as `model.type`, but warmstart methods have `BYOL`.
This condition is just for distinguishing warmstart methods (which don't use validation) from other models.
If you have better condition or it is required to modify config, please let me know.

Reply:
Warmstart conducts self-supervised learning during small epochs with target dataset to adapt model weights to target dataset.
For self-supervised learning, validation is not required, so validation set is not set in OTX.
You can refer https://github.com/openvinotoolkit/training_extensions/pull/1438/files#diff-3b8d1f587e0a91a74d0d736158cc1d7d76a706871315620662cb4e00afdfd7beR148-R155.